### ISSUE TITLE
loosing blocks intelligence after sudo su -

### ISSUE THREAD

**Issue by alexander-potemkin** [warpdotdev/Warp#2350]
### Describe the bug

- ssh to the host
- `sudo su -` there
- warp will loose block intelligence and will consider everything the rest as a single block inside `sudo su -`

P.S. Also is there a way for faster bugs reporting, without making me to fill in all of the details from my OS every time? There are quite a plenty of bugs there...

### To Reproduce

Provided above

### Expected behaviour

Recognize the blocks inside. OR, give me the option to use warp as a beautiful terminal - I like its colors and autocompletion (especially if you make it with Tab), AI integration is nice.

Otherwise I have to switch back to iTerm2.

### Operating System

None

### OS Version

Any

### Shell Version

Any

### Warp Version

Latest

### Does this block you from using Warp daily?

Yes

### Warp Internal (ignore): linear-label:b8107fdf-ba31-488d-b103-d271c89cac3e

None

**Comment by dannyneira** [comment]
Thanks for submitting! This is a known issue with Subshells not being supported at this time ( blocks/autocompletion/completions, etc ) See issue > #64 

As a workaround, you can disable the SSH Wrapper in Warp's `Settings > Features`, you'll lose blocks,etc. and all but the experience should be consistent across all SSH sessions.

p.s. regarding the OS/shell info requirement, we need this information for every request in order to know the details of whats being affected.

**Comment by alexander-potemkin** [comment]
> p.s. regarding the OS/shell info requirement, we need this information for every request in order to know the details of whats being affected.

yes, but there a lot of bugs to report and my sw configuration doesn't change during the day...

**Comment by kruegsche** [comment]
It's the same with me, I would like to continue using the fantastic features of warp even after changing the user with "su -".
After connecting via ssh, this initially works with the user used for the SSH connection. Unfortunately after a "su -" you can't use the great features anymore and you get an ordinary shell. Maybe there is a way I don't know or maybe you can update this in time.

I use the latest version of Warp (v0.2023.07.18.08.03.stable_00), MacOS Venture 13.4.1 and the remote servers are CentOS, AlmaLinux and OpenSuse.

**Comment by 0Knot** [comment]
In order to comply with security requirements, some servers use ordinary user password&key to log in. and no sudo permissions. Then use su -root to switch accounts. When you get here, you lose the block.hope to improve

**Comment by Emahhh** [comment]
Are there any news regarding this issue? I just experienced the same thing, and having support for this would be cool!
